functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -5,8 +5,9 @@
 import json
 import redis
 
-#Global value to be used to invalidate GET,PUT, and DELETE for Redis Cache
+#Global value to be used to invalidate GET, PUT, and DELETE for Redis Cache
 ALL_USERS_KEY = b'users:all'
```

Comment:
You don't need to use this - if you _want_, you can just save this as b'users:' and then concatenate each userId on it when you query or save to redis.

Reply:
This doesn't originate from my contribution to the code, but I can look into it

Reply:
Check out the way I implemented this in the UsersIdTask file - at the beginning of the request, I create the redis_key string like this: `redis_key = "users:" + str(userId)` and then pass it into your methods so they can use it when querying or clearing the cache.

Reply:
I created a bug issue (https://github.com/North-Seattle-College/ad440-winter2021-thursday-repo/issues/299) since this error was not due to changes from this PR.